web/src/components/Storyblok/StoryblokSectionNav/StoryblokSectionNav.tsx

```diff
@@ -0,0 +1,58 @@
+import React from "react";
+
+import { StackedNav, StackedNavLink } from "@nice-digital/nds-stacked-nav";
+
+type Link = {
+	id: number;
+	slug: string;
+	parent_id: number;
+	name: string;
+	is_folder: boolean;
+	is_startpage: boolean;
+	real_path: string;
+	childLinks?: Link[];
+};
+
+type StoryblokSectionNavProps = {
+	parentChildLinksTreeArray: Link[];
+	parentAndSiblingLinksElse: Link[];
+	slug: string;
+};
+
+export const StoryblokSectionNav = ({
+	parentChildLinksTreeArray,
+	parentAndSiblingLinksElse,
+	slug,
+}: StoryblokSectionNavProps): JSX.Element => {
+	const parentChildDataArray =
+		parentAndSiblingLinksElse?.length === 0
+			? parentChildLinksTreeArray
+			: parentAndSiblingLinksElse;
```

Comment:
Can't work out where the best place for this question is, but why do we need `parentAndSiblingLinksElse` to be distinct from `parentChildLinksTreeArray`?

Reply:
More generally, feel like we shouldn't be using `else` in var names as its hard to understand what it means without having to go and look into the code (think we use it here and in `parentelse`)